tests/testthat/test-utils.R

```diff
@@ -21,6 +21,11 @@ test_that("`try_exec_cmd()` returns stdout when command is available", {
   expect_equal(try_exec_cmd("echo", echo), echo)
 })
 
+test_that("`get_os()` returns a non-empty string", {
```

Comment:
Again, if you want to _really_ test it, you should try mocking, e.g. [this](https://testthat.r-lib.org/reference/local_mocked_bindings.html) to override state and test if under each distinct condition the function returns expected os. 
If you are only interested in `osx`, you then will have only two test cases.

Reply:
I've decided to remove it entirely. In short, I couldn't find a way to make testing worthwhile. I tried the following approaches:

1. The current implementation of `is_osx()` relies on `Sys.info()`, but mocking it would require setting `Sys.info <- NULL` somewhere in the package, which feels unconventional (see the "Base functions" section in `?local_mocked_bindings`).
2. A more straightforward approach would be `is_osx() <- try_exec_cmd("uname") == "Darwin"`, but this breaks several unrelated tests.